webpack/published-storybook-components.js

```diff
@@ -1,10 +0,0 @@
-const publishedStorybookComponents = {
-  // Components
-  "storybook/AlphaWarning": "storybook/components/alpha-warning/alpha-warning.tsx",
-  "storybook/Tip": "storybook/components/tip/tip.jsx"
```

Comment:
Apparently these are the only two components that were exported as `storybookComponents`, the rest is commented-out and I guess they will be moved to `vibe-storybook-components` as well which makes the build scripts redundant. I'm not sure who's consuming them, I looked up in GH search and couldn't find but I'm not 100% sure